dbmodel.sql

```diff
@@ -40,10 +40,12 @@ CREATE TABLE IF NOT EXISTS `card` (
   `age` TINYINT UNSIGNED DEFAULT NULL COMMENT '1 to 10, NULL for a special achievement',
   `faceup_age` TINYINT UNSIGNED DEFAULT NULL COMMENT 'The same as age, except Battleship Yamato is an 11 instead of 8 (dynamically populated)',
   `color` TINYINT UNSIGNED DEFAULT NULL COMMENT '0 (blue), 1 (red), 2 (green), 3 (yellow), 4 (purple) or NULL for a special achievement',
-  `spot_1` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on top-left, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock) or NULL for no icon (e.g. special achievement)',
-  `spot_2` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on bottom-left, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock) or NULL for no icon (e.g. special achievement)',
-  `spot_3` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on bottom-middle, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock) or NULL for no icon (e.g. special achievement)',
-  `spot_4` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on bottom-right, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock) or NULL for no icon (e.g. special achievement)',
+  `spot_1` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on bottom-left, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock), 7 (plus), 8 (flag), 9 (fountain), 10 (echo), 11 (left), 12 (right), 13 (up), 101-111 (1-11 bonus) or NULL for no icon (e.g. special achievement)',
```

Comment:
ooo I will have to increase the css for 7 through 13 and create something new for 101-111

Reply:
yeah, we need the sprites too.  Also, the format for the Cities cards is unique to all other Innovation cards so I imagine some CSS would need to change for that too.

Reply:
#375 has a list of stuff